daml-foundations/daml-ghc/test-src/DA/Test/ShakeIdeClient.hs

```diff
@@ -739,3 +740,12 @@ stressTests mbScenarioService = Tasty.testGroup "Stress tests"
 -- | Suppress unused binding warning in case we run out of tests for open issues.
 _suppressUnusedWarning :: ()
 _suppressUnusedWarning = testCaseFails `seq` ()
+
+platformSpecificMsg :: T.Text -> T.Text
+platformSpecificMsg msg = case os of
+    "mingw32" -> let
+                     repl '‘' = '`'
+                     repl '’' = '\''
+                     repl  c   = c
```

Comment:
I think this smart quote setting is something we should probably disable for everyone. I'm not sure I like it.

Reply:
Is there a setting for this (DAML/GHC?) or do you mean unifying them at DAML compiler level (not only in tests)?

Reply:
The smart quotes are controlled by `useUnicode` variable in DynFlags. As a result, it's not an OS determined variable, but something from sniffing the console. We should adjust the test case so it works with either, not on an OS specific basis.

If we're going to have smart quotes (which @cocreature seems to like) I'd rather turn on useUnicode everywhere, and then have a de-unicode pass for terminals that look like they might not be able to take it.

Reply:
Thanks, I moved the logic to `Testing.hs` where messages are compared and made the comparision quotes agnostic. No OS checks now.